www/includes/easyparliament/templates/html/people/mp_index.php

```diff
@@ -0,0 +1,226 @@
+                    <div class="people-list__your-mp__header">
+                      <p>
+                          Based on postcode <strong><?= $mp_data['postcode'] ?></strong>
+                          <a href="<?= $mp_data['change_url'] ?>">(Change postcode)</a>
+                      </p>
+                    <?php if ( isset($reps) ) { ?>
+                      <h3>Your <?= $rep_plural ?> are</h3>
+                    </div>
+                        <?php foreach ( $reps as $rep ) { ?>
+                    <a href="<?= $rep['mp_url'] ?>" class="people-list__person">
+                    <img class="people-list__person__image" src="<?= $rep['image'] ?>">
+                        <h2 class="people-list__person__name"><?= $rep['name'] ?></h2>
+                        <p class="people-list__person__memberships">
+                        <span class="people-list__person__constituency"><?= $rep['constituency'] ?></span>
+                        <span class="people-list__person__party <?= strtolower( $rep['party'] ) ?>"><?= $rep['party'] ?></span>
+                        </p>
+                    </a>
+                        <?php } ?>
+                    <?php } else { ?>
+                      <h3>Your <?= $rep_name ?> is</h3>
+                    </div>
+                    <a href="<?= $mp_data['mp_url'] ?>" class="people-list__person">
+                    <img class="people-list__person__image" src="<?= $mp_data['image'] ?>">
+                        <h2 class="people-list__person__name"><?= $mp_data['name'] ?></h2>
+                        <p class="people-list__person__memberships">
+                        <span class="people-list__person__constituency"><?= $mp_data['constituency'] ?></span>
+                        <span class="people-list__person__party <?= strtolower( $mp_data['party'] ) ?>"><?= $mp_data['party'] ?></span>
+                        </p>
+                    </a>
+                    <?php } ?>
+                </div>
+            </div>
+        </div>
+
+      <?php } else { ?>
+
+        <form action="/search/">
+            <div class="search-page__section search-page__section--search">
+                <div class="search-page__section__primary">
+                    <p class="search-page-main-inputs">
+                    <?php if ( $type == 'peers' ) { ?>
+                        <label for="find-mp-by-name-or-postcode">Find <?= $rep_plural ?> by name:</label>
+                    <?php } else { ?>
+                        <label for="find-mp-by-name-or-postcode">Find your <?= $rep_name ?> by name or postcode:</label>
+                    <?php } ?>
+                        <input type="text" class="form-control" name="q" id="find-mp-by-name-or-postcode">
+                        <button type="submit" class="button">Find</button>
+                    </p>
+                </div>
+            </div>
+        </form>
+
+      <?php } ?>
+
+        <div class="search-page__section search-page__section--results">
```

Comment:
The special handling for when DISSOLUTION_DATE is set (MPs) or no data (MSPs) has gone.